build.gradle

```diff
@@ -131,6 +131,8 @@ subprojects {
     }
 
     dokka {
+        jdkVersion = 8
```

Comment:
Simply squash this to the previous commit 407e793c14bb45701bb7e3ecdfc8384ba6f6bae2?

Reply:
I actually moved it out of the previous commit to avoid the "and" in the commit message :)